sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/MathExpressionsSuite.scala

```diff
@@ -963,4 +964,18 @@ class MathExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
     checkEvaluation(WidthBucket(5.35, 0.024, Double.NegativeInfinity, 5L), null)
     checkEvaluation(WidthBucket(5.35, 0.024, Double.PositiveInfinity, 5L), null)
   }
+
+  test("SPARK-45473: Non-constant scale to Round") {
```

Comment:
This is not needed since there are tests in ExpressionTypeCheckingSuite.scala.

Reply:
Removed. Thanks.